fpga/src/main/scala/arty/TestHarness.scala

```diff
@@ -29,7 +29,7 @@ class ArtyFPGATestHarness(override implicit val p: Parameters) extends ArtyShell
 
   val harnessClock = clock_32MHz
   val harnessReset = hReset
-  val success = false.B
+  val success = IO(Output(Bool()))
```

Comment:
Have you tried to rebuild the bitstream with this? Also, I would match what the Chipyard testharness does if this works in the bitstream (i.e. create the IO and assign it to `false.B` instead of floating).

Reply:
Good point. When building the bitstream, I get this error:
```ERROR: [DRC UCIO-1] Unconstrained Logical Port: 1 out of 10 logical ports have no user assigned specific location constraint (LOC). This may cause I/O contention or incompatibility with the board power or connectivity affecting performance, signal integrity or in extreme cases cause damage to the device or the components to which it is connected. To correct this violation, specify all pin locations. This design will fail to generate a bitstream unless all logical ports have a user specified site LOC constraint defined.
```
Which I am sure pertains to the success signal. I will work on this.